.github/workflows/ci.yml

```diff
@@ -0,0 +1,30 @@
+name: Build
+on: [push, pull_request]
+
+jobs:
+  linux:
+    runs-on: ubuntu-latest
+    env:
+      CC: gcc-10
+      CXX: g++-10
+    steps:
+      - uses: actions/checkout@v2
+      - run: cmake .
+      - run: cmake --build .
+
+  macos:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: cmake .
+      - run: cmake --build .
+
+  android:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: mkdir build && cd build
```

Comment:
it's fine either way, but any reason this is done in a separate directory, but not for the others?

Reply:
Good question. Probably because of some error I encountered while testing this jobs. I don't remember exactly.

I don't see anything weird in the build logs, so I don't think it matters much.